frontend/src/Pages/UserPage.module.scss

```diff
@@ -84,3 +85,42 @@
         margin-right: 4px;
     }
 }
+
+.active .tooltipText {
```

Comment:
Tooltip looks nice, but please consolidate the color and border thickness with other menus, like:
<img width="256" alt="image" src="https://github.com/spaceshelter/orbitar/assets/2865203/59fc62c8-662b-4005-9308-a1c76ff45810">

https://github.com/spaceshelter/orbitar/blob/3c8d12aae420a65adde04dd0f2c22732439e5ebf/frontend/src/Components/PostComponent.module.scss#L116

Reply:
Fixed 
![image](https://github.com/spaceshelter/orbitar/assets/3017441/273f7098-1aa6-4227-ad35-851ba0bddd0f)